types/lru-cache/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6","DOM"
```

Comment:
Need the type `AbortController` in TS scope, and adding `DOM`  here is the only way I know how to do that

Reply:
This seems like a lot to pull in for one type.  Maybe polyfill the bits you need like the underlying library [appears to](https://github.com/isaacs/node-lru-cache/blob/91c165d5318f54bd845db5c68338966f56d582d2/index.js#L8)?

Reply:
This library (`lru-cache`) passes the value of `signal` to a user-provided function. Developers can do whatever they want in those function, so the value needs to have the type of the the full `AbortSignal`, even if the `lru-cache` implementation doesn't need it. I can copy & paste the definition of AbortSignal from https://github.com/microsoft/TypeScript/blob/3ec1b0480834fcdb971b98d6d2bc12822d641208/lib/lib.dom.d.ts#L1921-L1931 but IMO most users will already have `DOM` in their TS projects. In the future we could have more lightweight alternatives like importing from `@types/dom` or `typescript/dom-modular`, but today the only option is either drag in the entire `DOM` or replicate the definition of `AbortSignal` so the users know what param they are getting in their functions.

Reply:
This value is used as a parameter for a function provided by the user, and the user will expect to see the full definition of `AbortSignal` for that parameter.

I changed how this works by importing `DOM` via a tripple-slash command instead of `compilerOptions`.

Reply:
If the user has the DOM types, they'll merge with your polyfill and they'll see the full definition.  If they don't, they probably shouldn't see the full definition.